src/test/regress/specs/isolation_data_migration.spec

```diff
@@ -0,0 +1,73 @@
+setup
+{
+    CREATE TABLE migration_table (test_id integer NOT NULL, data text);
+}
+
+teardown
+{
+    DROP TABLE migration_table;
+}
+
+session "s1"
+
+step "s1-begin"
+{
+    BEGIN;
+}
+
+step "s1-begin-serializable"
+{
+    BEGIN TRANSACTION ISOLATION LEVEL SERIALIZABLE;
+	SELECT 1;
```

Comment:
So `SELECT 1` is to get a fix snapshot? Becuase `SELECT 1` does not update anything, so I don't see the difference between serializable mode and read committed mode. Maybe we can talk more on this tomorrow morning :)